easybuild/easyblocks/m/mcr.py

```diff
@@ -119,10 +119,16 @@ def install_step(self):
 
     def sanity_check_step(self):
         """Custom sanity check for MCR."""
-        custom_paths = {
-            'files': [],
-            'dirs': [os.path.join(self.subdir, x, 'glnxa64') for x in ['runtime', 'bin', 'sys/os']],
-        }
+        if self.version == "2016b":
+            custom_paths = {
+                'files': [],
+                'dirs': [os.path.join(self.subdir, x, 'glnxa64') for x in ['bin', 'cefclient/sys/os']],
+            }
+        else:
+            custom_paths = {
+                'files': [],
+                'dirs': [os.path.join(self.subdir, x, 'glnxa64') for x in ['runtime', 'bin', 'sys/os']],
+            }
```

Comment:
@mboisson it's cleaner to only include the version-specific stuff in the if/else bodies:

```python
custom_paths = {
    'files': [],
    'dirs': [os.path.join(self.subdir, 'bin', 'glnxa64')],
}
if LooseVersion(self.version) >= LooseVersion('2016b'):
    custom_paths.append(os.path.join(self.subdir, 'cefclient', 'sys', 'os', 'glnxa64'))
else:
    custom_paths.extend([
        os.path.join(self.subdir, 'runtime, 'glnxa64'),
        os.path.join(self.subdir, 'sys', 'os', 'glnxa64'),
    ])
```